back/engines/free/seo/app/controllers/seo/application_controller.rb

```diff
@@ -28,12 +28,15 @@ def sitemap
       @ideas = Pundit.policy_scope(nil, Idea).select(:slug, :updated_at, :project_id).where(project_id: @projects.map(&:id))
     end
 
-    def robots; end
+    def robots
+      # Disable caching as an extra measure to prevent web cache poisoning
+      response.headers['Cache-Control'] = 'no-store'
+    end
```

Comment:
I think it's fine to keep caching if the content is no longer vulnerable to an attacker. Also, I'm not sure this will be fully respected by CloudFront. It will likely cache the content for the minimum TTL, which is set to 60 seconds for `robots.txt`.

<img width="954" height="218" alt="SCR-20250930-jaib" src="https://github.com/user-attachments/assets/457bf84e-468d-4632-b31f-a4a1f9128348" />
from https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html#ExpirationDownloadDist

Reply:
Good to know! I will reverse the caching part.